src/transfer/pages/swap/swap.tsx

```diff
@@ -148,7 +148,7 @@ const SwapPage: React.FC<Props> = observer((props: Props) => {
           </li>
           <li>
             <SwapNoticeLabel>
-              <T _str="* Maximum 100,000 NCG per day" _tags={transifexTags} />
+              <T _str="* Maximum 5,000 NCG per day" _tags={transifexTags} />
```

Comment:
```suggestion
              <T _str="* Maximum {max, number, integer} NCG per day" _tags={transifexTags} max={5000} />
```

This code will show the same message, but allows to change maximum value without causing to retranslate same pharse.

Reply:
I applied it in 203fb86. 🙏🏻